src/Organic/PageInjection.php

```diff
@@ -487,12 +537,39 @@ function loadScript(id, src) {
                     loadScript: loadScript
                 }
             })();
-            // TODO: delete after full migration to organic.splitTest
-            window.BVTests = window.organic.splitTest
         </script>
         <?php
     }
 
+    public function injectScriptTag( $id, $src, $modulesSrc = null ) {
+        if ( ! $modulesSrc ) {
+            wp_print_script_tag(
+                [
+                    'id' => $id,
+                    'src' => $src,
+                    'async' => true,
+                ]
+            );
+            return;
+        }
+        wp_print_script_tag(
```

Comment:
It might be helpful to clarify why both scripts need to be added in this case, e.g.:
```
// The script to be used if modules are supported.
wp_print_script_tag(...
);
// The script to be used if modules are not supported.
wp_print_script_tag(...
);
```